src/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const { wrap } = require('@adobe/openwhisk-action-utils');
+const { logger } = require('@adobe/openwhisk-action-logger');
+const { wrap: status } = require('@adobe/helix-status');
+const { epsagon } = require('@adobe/helix-epsagon');
+const indexHtml = require('./html_json.js');
+
+module.exports.main = wrap(indexHtml)
+  .with(epsagon)
+  .with(status)
+  .with(logger.trace)
```

Comment:
Until now, if an error occurred in `src/html_json.js` it would throw and the "parent" action (`helix-index-files`) would see that error. With `logger.trace` an error is caught - does that change the behaviour of `helix-index-files`?

Reply:
it should return a 500, like in https://github.com/adobe/helix-index-pipelines/blob/86f88fbb66c12171c7f2f46a72119a05eaf66c25/test/specs/hlx_up/notfound.json

Reply:
I added a post-deploy test for the 404 case:

https://github.com/adobe/helix-index-pipelines/pull/299/commits/2d2ab87ee96a052697fdb51942933d4baf21f56b